examples/install_service.rs

```diff
@@ -0,0 +1,35 @@
+#[cfg(windows)]
+extern crate windows_service;
+
+#[cfg(windows)]
+fn main() {
+    use std::ffi::OsString;
+    use windows_service::service::{
+        ServiceAccess, ServiceErrorControl, ServiceInfo, ServiceStartType, ServiceType,
+    };
+    use windows_service::service_manager::{ServiceManager, ServiceManagerAccess};
+
+    let manager_access = ServiceManagerAccess::CONNECT | ServiceManagerAccess::CREATE_SERVICE;
+    let service_manager = ServiceManager::local_computer(None::<&str>, manager_access).unwrap();
+    let service_info = ServiceInfo {
+        name: OsString::from("ping_service"),
+        display_name: OsString::from("Ping service"),
+        service_type: ServiceType::OwnProcess,
+        start_type: ServiceStartType::OnDemand,
+        error_control: ServiceErrorControl::Normal,
+        executable_path: ::std::env::current_exe()
+            .unwrap()
+            .with_file_name("ping_service.exe"),
```

Comment:
This example registers the other example as a service, so all three examples we have a connected.

Reply:
Do they have to be? It's nice with examples that are small and isolated and can be studied and understood without having to grasp something larger.

Reply:
I read the examples now. I feel they are indeed isolated and separated. One can understand `install_service.rs` without understanding `ping_service`. So it's not really a problem IMO.

But it might be good with a comment or two somewhere along the lines of "Installs a service, here we use the other example binary, `ping_service`, to showcase the installation, but it can be replaced with any binary that the service will be based on" or similar. Because as a new reader it can be hard to know what code is just "here is an example of what value one could put here" and "This value must be exactly this for it to work", and it's always good to explain those things IMO.

Reply:
OK. that makes sense. I will comment on that. Apparently I didn't want to bundle installation into the service example since one can do that on their own if needed + it saves a few dozen LOC and makes examples easier to follow.